### PR TITLE
use regionname in endpoint url instead of static value

### PR DESCRIPTION
**Pull request by sagarchalise**
The readme states to change `REGION` as well as `ENDPOINT` if not using `us-east-1` for region. This change uses the `AWS_SES_REGION_NAME` in `AWS_SES_REGION_ENDPOINT` instead of static value i.e. will use `mail.{AWS_SES_REGION_NAME}.amazonaws.com` instead of `mail.us-east-1.amazonaws.com` even if it is not set.
Just a small ease in my opinion.

### CODE REVIEW

File: tests/test_settings.py
Comment (pcraciunoiu) [inline tests/test_settings.py:39]: Seems a bit safer to check this is `eu-west-1` on the right side of the equality, but I'm OK with it as-is, too. What do you think?

**Review by pcraciunoiu — APPROVED** [review]
Thanks for contributing!

File: tests/test_settings.py
Comment (sagarchalise) [reply tests/test_settings.py]: @pcraciunoiu I was just following the code tests pattern that was present. But it does make sense to check `REGION_NAME` as `main.eu-west-1.amazonaws.com` rather than placeholder I guess. Thanks for the suggestion.

**Comment by pcraciunoiu** [comment]
Thanks @sagarchalise! I don't see a huge need to release this, on its own. Would rather wait until we have more features or fixes landed, but if you really want I could...

**Pull request merged**
